cmd/csi-resizer/main.go

```diff
@@ -43,36 +43,12 @@ var (
 	csiTimeout  = flag.Duration("csiTimeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
 	showVersion = flag.Bool("version", false, "Show version")
 
-	enableLeaderElection      = flag.Bool("leader-election", false, "Enable leader election.")
-	leaderElectionType        = flag.String("leader-election-type", "endpoints", "the type of leader election, options are 'endpoints' (default) or 'lease' (strongly recommended)")
-	leaderElectionIdentity    = flag.String("leader-election-identity", "", "Unique identity of this resizer. Typically name of the pod where the resizer runs.")
-	leaderElectionNamespace   = flag.String("leader-election-namespace", "kube-system", "Namespace where this resizer runs.")
-	leaderElectionRetryPeriod = flag.Duration("leader-election-retry-period", time.Second*5,
-		"The duration the clients should wait between attempting acquisition and renewal "+
-			"of a leadership. This is only applicable if leader election is enabled.")
-	leaderElectionLeaseDuration = flag.Duration("leader-election-lease-duration", time.Second*15,
-		"The duration that non-leader candidates will wait after observing a leadership "+
-			"renewal until attempting to acquire leadership of a led but unrenewed leader "+
-			"slot. This is effectively the maximum duration that a leader can be stopped "+
-			"before it is replaced by another candidate. This is only applicable if leader "+
-			"election is enabled.")
-	leaderElectionRenewDeadLine = flag.Duration("leader-election-renew-deadline", time.Second*10,
-		"The duration that non-leader candidates will wait after observing a leadership "+
-			"renewal until attempting to acquire leadership of a led but unrenewed leader "+
-			"slot. This is effectively the maximum duration that a leader can be stopped "+
-			"before it is replaced by another candidate. This is only applicable if leader "+
-			"election is enabled.")
+	enableLeaderElection    = flag.Bool("leader-election", false, "Enable leader election.")
+	leaderElectionNamespace = flag.String("leader-election-namespace", "Namespace where this resizer runs.")
```

Comment:
Did you test this code? The `flag.String` method needs three arguments but you only provides two, this will make the CI test failed.

Reply:
good catch, thanks